bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/EGateHandler.java

```diff
@@ -84,9 +84,15 @@ public void initialize() {
 
         if (host != null && port > 0) {
             // Create a socket to eGate
-            try (Socket localEgateSocket = new Socket(host, port)) {
+
+            InetSocketAddress socketAddress = new InetSocketAddress(host, port);
+            Socket localEgateSocket = new Socket();
+            try {
+                localEgateSocket.connect(socketAddress, SOCKET_TIMEOUT_SEC);
```

Comment:
I checked the connect doc and the timeout parameter is expected in milliseconds.
Please update the constant name or provide a milliseconds value.

Reply:
rewritten